files/es/web/api/batterymanager/dischargingtime/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: BatteryManager.dischargingTime
+slug: Web/API/BatteryManager/dischargingTime
+tags:
+  - API
+  - Apps
+  - Batería
+  - Battery
+  - Firefox OS
+  - Gecko DOM Reference
+  - WebAPI
+  - aplicaciones
+translation_of: Web/API/BatteryManager/dischargingTime
+browser-compat: api.BatteryManager.dischargingTime
+---
+
+{{deprecated_header}}{{APIRef("Battery API")}}
+
+Indíca la cantidad de tiempo, en segundos,
+que restan antes de que la batería se descargue completamente.
+
+> **Nota:** Incluso si el tiempo devuelto es exacto al segundo,
+> los navegadores lo redondean a valor más alto.
+> intervalo (típicamente a los 15 minutos más cercanos) por razones de privacidad.
+
+## Sintaxis
+
+```js
+var time = battery.dischargingTime;
+```
+
+Valor de retorno, `time` es el tiempo restante en segundos antes que la `batería`,
+que es un objeto {{domxref("BatteryManager")}},
+esté completamente descargada y el sistema se suspenda.
+Este valor es [`Infinito`](/es/docs/Web/JavaScript/Reference/Global_Objects/Infinity)
+Si la batería se está cargando en vez de descargando,
+o si el sistema es incapaz de calcular el tiempo de descarga.
+
+## Ejemplo
+
+### Contenido HTML
+
+```html
+<div id="dischargingTime">(tiempo de descarga desconocido)</div>
+```
+
+### Contenido JavaScript
+
+```js
+navigator.getBattery().then(function (battery) {
+  var time = battery.dischargingTime;
+
+  document.querySelector("#dischargingTime").textContent = battery.dischargingTime;
+});
+```
+
+{{ EmbedLiveSample('Ejemplo', '100%', 30) }}
+
+## Especificaciones
+
+{{Specifications}}
+
+## Compatibilidad del navegador
+
+{{Compat}}
+
+## Ver también
+
+- {{domxref("BatteryManager")}}
+- {{domxref("Navigator.getBattery")}}
```

Comment:
Nitpicking here — there should be a newline at the very end of every file

```suggestion
- {{domxref("Navigator.getBattery")}}

```

Reply:
done, (even the English version doesn't have it)

Reply:
It seems that, for some reason, newlines at the end of files don't show in GitHub sometimes, so it's possible that the newline is in the en-US version but the interface doesn't show it

Reply:
Okay! another new thing to keep in mind :thought_balloon: